resources/views/components/forum/post.blade.php

```diff
@@ -11,6 +11,27 @@ class="post__datetime"
         >
             {{ $post->created_at?->diffForHumans() }}
         </time>
+
+        @if ($post->updated_at > $post->created_at)
+            <span class="post__edited">
+                <i class="{{ config('other.font-awesome') }} fa-minus fa-xs"></i>
+                edited
+                <i class="{{ config('other.font-awesome') }} fa-caret-down"></i>
+                <div class="post__edited-dropdown">
+                    {{ $post->updated_at?->diffForHumans() }}
+                    by
+                    <a
+                        class="user-tag__link {{ $post->updatedBy->group->icon }}"
+                        href="{{ route('users.show', ['user' => $post->updatedBy]) }}"
+                        style="color: {{ $post->updatedBy->group->color }}"
+                        title="{{ $post->updatedBy->group->name }}"
```

Comment:
This breaks for posts that have an `updated_at` timestamp already, but have a null `updated_by`.

`Attempt to read property "group" on null`

Reply:
Fixed